docs(readme): add PageSpeed Green to to related projects (#10018)
GoogleChrome/lighthouse: readme.md

* **[Lighthouse Keeper](https://lighthouse-keeper.com/)** - Lighthouse Keeper monitors your pages' Lighthouse scores and notifies you if they drop below your thresholds. Lighthouse Keeper is a free service that monitors up to 3 URLs once per day.

> * **[PageSpeed Green](https://pagespeed.green/)** - New web performance monitoring service for web pages. For now, you can track PageSpeed performance indicators for unlimited domains and web pages. Coming, running in different regions and CD/CI integration. Powered By Google Lighthouse and Puppeteer. You can easily start using PageSpeed Green with 30-day trial.

Reviewer: ```suggestion
* **[PageSpeed Green](https://pagespeed.green/)** - Web performance monitoring service that tracks Lighthouse performance indicators on unlimited domains and web pages.  PageSpeed Green is a paid product with a 30-day trial.
```

maybe come back to add the next sentence about different regions and CI/CD once it's implemented :)

Author: @patrickhulce Thanks for review! Yes, it my own infrastructure. Yes, good advise about screenshots. Ok, I'll remove info about future's features.

Reviewer: did you dislike the other edits? the other paid products follow the model of `X is a paid product with a Y-day trial.`

Author: I do not like to be similar :-) Ok, I changed to the existing approach
